torch/_inductor/codecache.py

```diff
@@ -1257,6 +1256,13 @@ def _check_can_cache(gm: torch.fx.GraphModule) -> None:
         Check some conditions that would preclude caching and raise BypassFxGraphCache
         to bypass in case caching is not possible.
         """
+        # TODO: https://github.com/pytorch/pytorch/issues/130772
+        if (
+            config.post_grad_custom_pre_pass is not None
+            or config.post_grad_custom_post_pass is not None
+        ):
+            raise BypassFxGraphCache("Post grad custom passes unsupported")
```

Comment:
it'd be a couple more lines of code to check if `config.post_grad_custom_post_pass.get("fx_graph_cache_key")` exists so users could unblock themselves.  i think we should add it, or a variant